pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -768,14 +768,10 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 
 	if vmi.Spec.Domain.CPU != nil {
 		// Set VM CPU cores
-		domain.Spec.CPU.Topology = &CPUTopology{
-			Sockets: 1,
-			Cores:   calculateRequestedVCPUs(vmi),
-			Threads: 1,
-		}
+		domain.Spec.CPU.Topology = getCPUTopology(vmi)
```

Comment:
The only change that was needed here was to increase `Sockets` instead of `Cores`

Reply:
Making it confusing and preventing the support of full topology in the future..